parsel/selector.py

```diff
@@ -175,8 +175,11 @@ def xpath(self, query):
             return self.selectorlist_cls([])
 
         try:
-            result = xpathev(query, namespaces=self.namespaces,
-                             smart_strings=self._lxml_smart_strings)
+            nsp = dict(self.namespaces)
+            nsp.update(kwargs.pop('namespaces', {}))
```

Comment:
why not adding `namespaces` as a keyword function argument with default value? 

btw, if we are going to mix xpath kwargs with function arguments, lets use a prefix like `parsel_namespaces` or a single argument for all xpathev kwargs.